controllers/chat/ChatController.js

```diff
@@ -91,6 +91,8 @@ module.exports = {
         {last_message_at: -1}
       ]);
 
+      // const createdChannel = await cha
+
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The commented out code on line 94 seems to be incomplete and doesn't provide any context. If it's not needed, it should be removed to keep the code clean.
```diff
- // const createdChannel = await cha
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->